From 49c33a13a53f7454dddaf87e61d04c11c7d9cfd9 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Wed, 18 Feb 2015 15:53:43 +0000
Subject: [PATCH 1/4] Include stateless configuration.

---
 conf.d/50-user.conf | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/conf.d/50-user.conf b/conf.d/50-user.conf
index 681ed6d..9a24f58 100644
--- a/conf.d/50-user.conf
+++ b/conf.d/50-user.conf
@@ -6,12 +6,18 @@
   </its:rules>
 
   <description>Load per-user customization files</description>
+	<!-- global OS wide defaults -->
+	<include ignore_missing="no">/usr/share/defaults/fonts/conf.d</include>
+	<include ignore_missing="no">/usr/share/defaults/fonts/fonts.conf</include>
 	<!--
 	    Load per-user customization files where stored on XDG Base Directory
 	    specification compliant places. it should be usually:
 	      $HOME/.config/fontconfig/conf.d
 	      $HOME/.config/fontconfig/fonts.conf
 	-->
+	<include ignore_missing="yes">/etc/fonts/conf.d</include>
+	<include ignore_missing="yes">/etc/fonts/fonts.conf</include>
+
 	<include ignore_missing="yes" prefix="xdg">fontconfig/conf.d</include>
 	<include ignore_missing="yes" prefix="xdg">fontconfig/fonts.conf</include>
 	<!-- the following elements will be removed in the future -->
-- 
2.21.0

