From 6d86e5a6b1b684ef8ef765a44d511a184e8e7c1d Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Wed, 18 Feb 2015 15:53:43 +0000
Subject: [PATCH 1/4] Include stateless configuration.

---
 conf.d/50-user.conf | 6 ++++++
 src/fcinit.c        | 2 ++
 2 files changed, 8 insertions(+)

diff --git a/conf.d/50-user.conf b/conf.d/50-user.conf
index 681ed6d..9a24f58 100644
--- a/conf.d/50-user.conf
+++ b/conf.d/50-user.conf
@@ -6,12 +6,18 @@
   </its:rules>
 
   <description>Load per-user customization files</description>
+	<!-- global OS wide defaults -->
+	<include ignore_missing="no">/usr/share/defaults/fonts/conf.d</include>
+	<include ignore_missing="no">/usr/share/defaults/fonts/fonts.conf</include>
 	<!--
 	    Load per-user customization files where stored on XDG Base Directory
 	    specification compliant places. it should be usually:
 	      $HOME/.config/fontconfig/conf.d
 	      $HOME/.config/fontconfig/fonts.conf
 	-->
+	<include ignore_missing="yes">/etc/fonts/conf.d</include>
+	<include ignore_missing="yes">/etc/fonts/fonts.conf</include>
+
 	<include ignore_missing="yes" prefix="xdg">fontconfig/conf.d</include>
 	<include ignore_missing="yes" prefix="xdg">fontconfig/fonts.conf</include>
 	<!-- the following elements will be removed in the future -->
diff --git a/src/fcinit.c b/src/fcinit.c
index 5831a19..fc93c4b 100644
--- a/src/fcinit.c
+++ b/src/fcinit.c
@@ -45,6 +45,8 @@ FcInitFallbackConfig (const FcChar8 *sysroot)
 	"  <dir prefix=\"xdg\">fonts</dir>" \
 	"  <cachedir>" FC_CACHEDIR "</cachedir>" \
 	"  <cachedir prefix=\"xdg\">fontconfig</cachedir>" \
+	"  <include ignore_missing=\"no\">/usr/share/defaults/fonts/conf.d</include>" \
+	"  <include ignore_missing=\"no\">/usr/share/defaults/fonts/fonts.conf</include>" \
 	"  <include ignore_missing=\"yes\" prefix=\"xdg\">fontconfig/conf.d</include>" \
 	"  <include ignore_missing=\"yes\" prefix=\"xdg\">fontconfig/fonts.conf</include>" \
 	"</fontconfig>";
-- 
2.21.0

